docs/token_authorization_api.md

```diff
@@ -2,6 +2,7 @@
 
 TorchServe now enforces token authorization by default
 
+TorchServe introduces a new feature called token-authorization which requires the correct token to be provided when calling an API. This is a security feature which addresses the concern of unwanted API calls. This is applicable in the scenario where an unauthorized user may try to access a running TorchServe server. The default behavior is to enable this feature which creates a key file with the appropriate tokens to be used for API calls. Users can disable this feature to prevent token authorization from being required for API calls.
```

Comment:
We introduced it before, now we're enabling it as a default. Add the parameter how people can disable it but warn of the consequences.